src/release/triage-procedure.md

```diff
@@ -130,3 +130,11 @@ each PR:
 
 Once you are done triaging PRs, post your report in the topic for the current
 week's triage in the `#t-release/triage` Zulip stream.
+the topic should have a name like `YYYY-MM-DD to YYYY-MM-DD`.
+Note that this uses a monday-sunday week.
+if a topic does not exist, you can generate it's title with
+the following bash oneliner (requires GNU date):
+`echo "$(date -I --date="$([ "z$(date +%a)" = "zMon" ] && echo 'today' || echo 'last monday')") to $(date -I --date="$([ "z$(date +%a)" = "zSun" ] && echo 'today' || echo 'next sunday')")"`
```

Comment:
I'd probably improve a little bit the formatting because the rendered markdown is a bit cluttered
![grafik](https://github.com/user-attachments/assets/40a2c577-bddd-4c7d-8d92-7a5c57eccd53)

Reply:
Suggestion:
```
echo "$(date -I --date="$([ "z$(date +%a)" = "zMon" ] && \
    echo 'today' || echo 'last monday')" \
    to $(date -I --date="$([ "z$(date +%a)" = "zSun" ] && \
    echo 'today' || echo 'next sunday')")"
```

Reply:
No can do, this one-liner is very nested and you can't use `\` to break within the interpolation AFAICT. Just make it a separate code block but kept the one-liner.